attributes/default.rb

```diff
@@ -48,19 +48,18 @@
 default['abiquo']['tomcat']['ajp-port'] = 8010
 default['abiquo']['tomcat']['wait-for-webapps'] = false
 
+# Override the Apache proxy configuration
+override['apache']['proxy']['order'] = "allow,deny"
+override['apache']['proxy']['deny_from']  = "none"
+override['apache']['proxy']['allow_from'] = "all"
```

Comment:
Now that I see this, can we change all properties marked as "override" to "default"?